doc/set_fees_process.md

```diff
@@ -0,0 +1,40 @@
+# Setting fees process
+
+It will be explained on an example of Indy CLI.
+
+### Prerequisites
+
+* libindy
+* Indy CLI
+* libsovtoken
+* Wallet with Trustee DID
+* Created pool in Indy CLI
+
+[Small guide](https://github.com/hyperledger/indy-sdk/tree/master/doc/design/001-cli#commands) to Indy CLI commands: 
+
+### Creating a SET_FEES transaction.
+
+* Open `indy-cli`
+* Open wallet: `wallet open <wallet_name> key=<wallet_encryption_key>`
+* Load libsovtoken: `load-plugin library=libsovtoken.[so|dll] initializer=sovtoken_init`
+* Make a transaction with a command `ledger set-fees-prepare payment_method=sov fees=NYM:1` to set fees to 1 sovatom for NYM
+
+Indy CLI will print the transaction after these steps.
+
+### Putting your signature on a prepared request
+
+* Open `indy-cli`
+* Load libsovtoken: `load-plugin library=libsovtoken.[so|dll] initializer=sovtoken_init`
+* Open your wallet: `wallet open <name_of_wallet> key=<wallet_encryption_key>`
+* Use your did: `did use <your_did>`
+* Sign the transaction you created or received: `ledger sign-multi txn=<transaction_json>`
+
+Indy CLI will print the transaction with your signature after these steps.
```

Comment:
Maybe a comment here about needing multiple signatures on the transaction before it is sent to the ledger?

Reply:
And should the directions say anything about how to get the the transaction from one trustee to the next?

Reply:
I don't think it is needed, because trustees should already know what is needed for such transaction to be written. We are only covering technical details (how to put your signature, how to build a request, how to send it to the ledger)
For the second one, I can just mention that they should share signed transaction to other trustees, but again, they should be aware of the high level process.

Reply:
Added some comments. And one more point about getting the transaction from one trustee to another -- do we even know this?